cylc/flow/host_select.py

```diff
@@ -332,11 +344,11 @@ def visit(self, node):
         # variables
         ast.Name, ast.Load, ast.Attribute, ast.Subscript, ast.Index,
         # opers
-        ast.BinOp, ast.operator,
+        ast.BinOp, ast.operator, ast.UnaryOp, ast.unaryop,
```

Comment:
This adds the required oppererators.

See [ast](https://docs.python.org/3/library/ast.html?highlight=ast#abstract-grammar) for more info.

Reply:
What is the difference between `UnaryOp` and `unaryop`?